ci/test-opentxs

```diff
@@ -14,13 +15,15 @@ if [ ! -d "${WORK}" ]; then
     exit 1
 fi
 
-if [ "${JOBS}" == "" ]; then
-    echo "Parallel test jobs not set"
+if [ "${CTEST_PARAMS}" == "" ]; then
+    echo "ctest params not set"
     exit 1
 fi
 
 set -e
 
 cd "${WORK}"
-cmake --install .
-ctest -j "${JOBS}" --repeat until-pass:5 --schedule-random
+
+echo ${CTEST_PARAMS}
+#cmake --install .  #no need to install
```

Comment:
In general lines that are unnecessary should just be deleted and not commented out unless maybe there is a plan to bring them back some day.